README.md

```diff
@@ -69,22 +69,22 @@ Getting started is very easy:
 Install `foundryup`:
 
 ```
-curl -L https://foundry.paradigm.xyz | bash
+curl -L https://raw.githubusercontent.com/paritytech/foundry-polkadot/refs/heads/master/foundryup/install | bash
 ```
 
 Next, run `foundryup`.
 
-It will automatically install the latest version of the precompiled binaries: [`forge`](#forge), [`cast`](#cast), [`anvil`](#anvil), and [`chisel`](#chisel).
+It will automatically install the latest version of the precompiled binaries: [`forge`](#forge), [`cast`](#cast).
 
 ```
-foundryup
+foundryup --nightly
```

Comment:
why nightly? I assume we want to install pre-release (release candidate). Foundryup does not allow for pre-release installations?

Reply:
fixed